forge-gui/res/cardsfolder/upcoming/minthara_merciless_soul.txt

```diff
@@ -0,0 +1,10 @@
+Name:Minthara, Merciless Soul
+ManaCost:2 W B
+Types:Legendary Creature Elf Cleric
+PT:2/2
+K:Ward:X
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield |  ValidTgts$ Permanent.YouOwn | Origin$ Battlefield | Destination$ Graveyard |  Execute$ TrigExperience | TriggerDescription$ At the beginning of your end step, if a permanent you controlled left the battlefield this turn, you get an experience counter.
```

Comment:
I think this will trigger on your end step even if a permanent you controlled _didn't_ leave the battlefield that turn

Reply:
`Origin$` and `Destination$` should not do anything in a Phase trigger

Reply:
I think instead of those, just use `Revolt$ True` and this should work!

Reply:
Also: `|..Execute$` -> `|.Execute$` (. = space)

Reply:
also remove ValidTgts$ - you are not targeting anything

Reply:
Does this work well? If so, feel free to resolve this conversation

Reply:
It works, thanks!